### DISCUSSION TITLE
TinyMCE color picker dropdown appears off-screen

### DISCUSSION THREAD

**Discussion by abdessamadpas** [tinymce/tinymce#9673] [General]
I still have this bug and I've searched everywhere for a solution but haven't had any luck 😔. I'm using Angular 15 with @tinymce/tinymce-angular version 7.0.0. Any ideas on how to resolve it or get closer to a solution?
 this bug while scrolling
![image](https://github.com/tinymce/tinymce/assets/53188247/32619478-3bc6-43bf-a3df-83ed49536dc7)

**Comment by lostkeys** [comment]
@abdessamadpas Since there are not much to go on and troubleshoot, I'll convert your issue to a discussion topic for others to chime in with their experience. Please feel free to submit a proper bug report with a demo reproducing the issue.

**Comment by TheSpyder** [comment]
The [`ui_mode`](https://www.tiny.cloud/docs/tinymce/latest/ui-mode-configuration-options/#ui_mode) setting should solve this for you.
